include/swift/AST/TypeAttr.def

```diff
@@ -64,6 +64,7 @@ SIMPLE_TYPE_ATTR(_local, Local)
 SIMPLE_TYPE_ATTR(_noMetadata, NoMetadata)
 TYPE_ATTR(_opaqueReturnTypeOf, OpaqueReturnTypeOf)
 TYPE_ATTR(isolated, Isolated)
+SIMPLE_TYPE_ATTR(_addressable, Addressable)
```

Comment:
Is this a type attribute or does it make sense to make this a parameter attribute? e.g.
```swift
func foo(_: @_addressable Int)
// vs
func foo(@_addressable _: Int)
```